src/Toml/BiMap.hs

```diff
@@ -133,6 +139,26 @@ _Double = mkAnyValueBiMap matchDouble Double
 _Text :: BiMap Text AnyValue
 _Text = mkAnyValueBiMap matchText Text
 
+_ZonedTime :: BiMap ZonedTime AnyValue
+_ZonedTime = mkAnyValueBiMap (getTime . matchDate) (Date . Zoned)
+  where getTime (Just (Zoned z)) = Just z
```

Comment:
You can write this as `matchDate >=> getTime` to avoid pattern-matching on `Just` in `getTime` function